alacritty.yml

```diff
@@ -22,6 +22,20 @@
   # check the local terminfo database and use `alacritty` if it is
   # available, otherwise `xterm-256color` is used.
   #TERM: alacritty
+  #
+  # WINIT_X11_SCALE_FACTOR
+  #
+  # This value is used to control scale factor used by terminal on X11, by
+  # default it automatically detects scale factor of each monitor and apply
+  # it dynamically by using XRandr. Supported values `randr` or floating point
+  # values, like `"1.0"`.
+  #WINIT_X11_SCALE_FACTOR: randr
+  #
+  # WINIT_WAYLAND_CSD_THEME
+  #
+  # This value is used to pick between Wayland client side decorations variant.
+  # For now only `dark` and `light` are supported.
+  #WINIT_WAYLAND_CSD_THEME: light
```

Comment:
Should we maybe translate this to an Alacritty option if it's relevant enough to show up in our configuration file?

Reply:
As an option for config reload? I think we can do that, just not sure that you'd want to.

Reply:
I've decided to unify it a bit.